Bluepill-cli/Bluepill-cli/Simulator/BPSimulator.m

```diff
@@ -170,7 +170,34 @@ - (NSRunningApplication *)findSimGUIApp {
     return nil;
 }
 
+- (void)addVideoToSimulator {
```

Comment:
Small detail but since this is adding multiple videos, shouldn't it be called `addVideosToSimulator`?